src/cascadia/TerminalSettingsModel/CascadiaSettings.h

```diff
@@ -96,7 +96,7 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
         void _addOrMergeUserColorScheme(const winrt::com_ptr<implementation::ColorScheme>& colorScheme);
         void _executeGenerator(const IDynamicProfileGenerator& generator);
 
-        std::unordered_set<std::wstring_view> _ignoredNamespaces;
+        std::unordered_set<winrt::hstring, til::transparent_hstring_hash, til::transparent_hstring_equal_to> _ignoredNamespaces;

```

Comment:
can we get transparency from `unordered_set` with C++23 without our own hash/equal types?

Reply:
> Expecting good, novel things to come from C++

A fellow fantasy genre enjoyer!

Jokes aside, as far as I know, the only thing that C++23 adds are more heterogenous overloads, but it remains opt-in so as to not break existing code. It would also not help us here, because just like C++, cppwinrt is also frozen in time and it's what would require the `is_transparent` attribute on its hash/equal_to structs ([here](https://github.com/microsoft/cppwinrt/blob/f9ec1986083a70d2f99d726b06b00a38cb2d1054/strings/base_std_hash.h#L37-L43)).

Reply:
🥲